content/docs/sync-async-boundary.md

```diff
@@ -0,0 +1,43 @@
+# Sync-async boundary
+
+If my experience in the IT industry has taught me anything it would be that drawing boundaries is the most important part of the design process. Boundaries are essential for understanding and communicating the design. The sync-async boundary is an example of af a boundary that is useful when designing distributed systems.
+
+## Sync
+
+The most popular medium for building synchronous systems is HTTP. At the topmost level the user's action is transformed into a HTTP request by the code running in the browser and submitted to one of the backend components for processing. The response carries the result of the action for rendering on screen. Internally the component that does the processing can delegate part of the work to other components. It does so by sending HTTP its own requests and collecting responses. These other components can do the same. This can result in the request "tree" that can span dozens of components.
```

Comment:
```suggestion
The most popular medium for building synchronous systems is HTTP. In the outermost layer, the user's action is transformed into an HTTP request by the code running in the browser and submitted to one of the backend components for processing. The response carries the result of the action for rendering on screen. Internally the component that does the processing can delegate part of the work to other components. It does so by sending its own requests and collecting responses. These other components can do the same. This can result in the request "tree" that can span dozens of components.